src/main/scala/rocket/Multiplier.scala

```diff
@@ -6,10 +6,10 @@ package freechips.rocketchip.rocket
 import chisel3._
 import chisel3.util.{Cat, log2Up, log2Ceil, log2Floor, Log2, Decoupled, Enum, Fill, Valid, Pipe}
 import freechips.rocketchip.util._
-import ALU._
+import freechips.rocketchip.config.Parameters
 
-class MultiplierReq(dataBits: Int, tagBits: Int) extends Bundle {
-  val fn = Bits(SZ_ALU_FN.W)
+class MultiplierReq(dataBits: Int, tagBits: Int)(implicit val p: Parameters) extends Bundle with HasRocketCoreParameters {
+  val fn = Bits(alufn.SZ_ALU_FN.W)
```

Comment:
Would `class MultiplierReq(dataBits: Int, tagBits: Int, alufn: ALUFN = ALUFN())` work?

Reply:
Done.

Reply:
Please provide default for alufn to be the base, this makes migration much easier.

Reply:
Done.